### DISCUSSION TITLE
Constant power & tracing on a different generation

### DISCUSSION THREAD

**Discussion by jeffreyspaan** [accel-sim/accel-sim-framework#257] [Q&A]
Hi,

1. All my simulations using Accel-Sim (SASS mode) with AccelWattch report an avg/max/min constant power of 0 in the `accelwattch_power_report.log`. Is this expected? From the source code it seems like it should be just copied over from the `constant_power` parameter in the `accelwattch_sass_sim.xml`.
2. I use traces generated on an Ampere GPU to simulate a Turing configuration (2060_S). All simulations seem to work fine. Should I be concerned about any inaccuracies?
3. What is the purpose of the `dvfs_enabled` parameter? From what I gathered (#227), Accel-Sim doesn't support dynamically changing the frequency, so what is the power model modelling? 

Thanks in advance.

**Comment by JRPan** [comment]
1. Not sure about that. You may want to wait for @VijayKandiah's answer
2. No comment on that. But I would not do that. I'm actually surprised it finished. Generally, it would fail because of SASS opcode mismatch.
3. It's just controlling the formula for calculating power. Not actually changing the frequency. The frequency is fixed, and the parameter is controlling what the voltage is.
